test/e2e/node/OWNERS

```diff
@@ -0,0 +1,3 @@
+reviewers:
+  - koordlet-approvers
+  - runtime-proxy-approvers
```

Comment:
also need add `approvers` section.
```yaml
approvers:
   - koordlet-approvers
   - runtime-proxy-approvers
```

Reply:
done. change the approvers as the koordlet's and the slo-controller's